src/Toaster/Toaster.tsx

```diff
@@ -43,7 +64,7 @@ export default class Toaster extends React.Component {
   }
 
   componentDidMount() {
-    const node = this.props.attachTo ? this.toaster : ReactDOM.findDOMNode(this).parentNode;
+    const node = this.props.attachTo ? this.toaster! : ReactDOM.findDOMNode(this).parentElement!;
```

Comment:
both `!` are safe: if `this.props.attachTo` is `true`, then `toaster` has been initialized in `componentWillMount` and `parentElement` is known be not null.